src/NexusMods.Icons/SimpleVector/Control/SimpleVectorIcon.cs

```diff
@@ -0,0 +1,39 @@
+using Avalonia;
+using Avalonia.Controls.Documents;
+using Avalonia.Media;
+
+namespace NexusMods.Icons.SimpleVector.Control;
+
+/// <summary>
+///     A wrapper for <see cref="Avalonia.Controls.Image"/> designed to work with <see cref="SimpleVectorIconImage"/>.
+/// </summary>
+public class SimpleVectorIcon : Avalonia.Controls.Image
```

Comment:
Why do we need this wrapper?

Reply:
It adds a Foreground property, so the control works with our existing styles.